example_reducing_utils_test.go

```diff
@@ -153,3 +153,29 @@ func ExamplePPmt_loan() {
 	// period:23 principal:-4846
 	// period:24 principal:-4918
 }
+
+// If an investment has a 6% p.a. rate of return, compounded annually, and you wish to possess ₹ 1,49,716 at the end of 10 peroids while providing ₹ 10,000 per period,
+// how much should you put as your initial deposit ?
+func ExamplePv() {
+	rate := 0.06
+	nper := int64(10)
+	payment := float64(-10000)
+	fv := float64(149716)
+	when := paymentperiod.ENDING
+
+	pv := gofinancial.Pv(rate, nper, payment, fv, when)
+	fmt.Printf("pv:%v", math.Round(pv))
+	// Output:
+	// pv:-10000
+}
+
+// Given a rate of 0.281 per period and initial deposit of 100 followed by withdrawls of 39, 59, 55, 20.
```

Comment:
```suggestion
// Given a discount rate of 28.1% per period and initial deposit of 100 followed by withdrawls of 39, 59, 55, 20.
```
yeah, rest looks good. just one more thing. This 28% discount rate is not plausible. Write an example with a realistic discount rate, something like 8%. refer [this](https://github.com/numpy/numpy-financial/blob/master/numpy_financial/_financial.py#L836)